tests/e2e/env/setup.sh

```diff
@@ -105,12 +105,6 @@ if [[ -z $CI ]]; then
 	docker-compose -f "$E2E_ROOT"/env/docker-compose.yml up --build --force-recreate -d phpMyAdmin
 fi
 
-if [[ -n $CI ]]; then
```

Comment:
@htdat Like you mentioned, on previous modifications, I added this condition to allow xDebug for debugging locally. I'm OK with this change, let see if other have more queries.